docs/source/datasets.rst

```diff
@@ -307,71 +307,62 @@ When retrieving a single dataset record, the following version-related fields ar
 Using the identifiers provided by the above fields, it's possible to retrieve information about a specific dataset version using the standard datasets API ``GET /rest/datasets/<pid>``.
 
 
+.. only:: local_development or test
 
-Datasets with access type of permit
----------------------------------------
+    Using REMS
+    -----------
 
 
-**General**
+    REMS can be used to give access for downloading dataset files to individual users. When dataset access is REMS managed dataset owner can decide which users are able to download the files affiliated to the dataset.
 
-Downloads of files affiliated to dataset can be restricted individually. This means that selected user can decide who can get download access to the files in dataset. When this kind of functionality is wanted, dataset must have the following information:
+    To enable REMS, set ``access_type`` to ``permit`` and ensure that dataset belongs to IDA catalog and has at least one license defined. You can enable REMS when creating a new dataset or later while updating an existing dataset.
 
-* ``access_type`` must be set to permit
-* Dataset must have at least one license
-* Query parameter ``access_granter`` on the POST/PUT/PATCH request which changes the ``access_type`` to permit. The parameter contains the information about the user who is deciding whether to grant the download access or not. More information of access_granter below.
 
-When this information is available, Metax creates an entity to REMS (Resource Entitlement Management System) which is used to manage the granting process. The entity in REMS is called ``catalogue-item`` and it contains information about the access granter, first described ``license``, ``preferred_identifier`` and the ``metadata_owner_org`` of the dataset.
+    **Changing access type**
 
+    When ``access_type`` is set to ``permit`` dataset downloads are managed by REMS. If this functionality is no longer wanted, simply changing the ``access_type`` to any other access type disables REMS for the dataset. Example of defining permit access type:
```

Comment:
Add comma: ``When access_type is set to permit, dataset downloads are managed by REMS.``